build/secondary/testing/benchmark/BUILD.gn

```diff
@@ -0,0 +1,57 @@
+# Copyright 2017 Google, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+static_library("benchmark") {
+  if (current_toolchain == host_toolchain) {
+    defines = [ "HAVE_STD_REGEX" ]
+  }
+
+  include_dirs = [
+    "//third_party/benchmark/include",
+  ]
+
+  sources = [
+    "//third_party/benchmark/src/arraysize.h",
```

Comment:
This file is not in the //third_party/benchmark directory. Is there another way to reference those files without the prefix?